infra/modules/azure_core_infra/example/complete/locals.tf

```diff
@@ -9,10 +9,11 @@ locals {
   }
 
   tags = {
-    CreatedBy   = "Terraform"
-    Environment = "Dev"
-    Owner       = "DevEx"
-    Source      = "https://github.com/pagopa/dx/modules/azure_core_infra/examples/complete"
-    CostCenter  = "TS700 - ENGINEERING"
+    CostCenter     = "TS000 - Tecnologia e Servizi"
```

Comment:
issue: is it possible to remove unrelated changes from this PR? we can fix these with anoher one and make this dependant